src/IdentityManager/WebApi/Controllers/AccountController.cs

```diff
@@ -0,0 +1,186 @@
+        }
+
+        /// <summary>
+        /// ForgetPassword
+        /// </summary>
+        /// <param name="email"></param>
+        /// <returns></returns>
+        [HttpPost("ForgetPassword")]
+        public async Task<IActionResult> ForgetPassword(string email)
+        {
+            if (string.IsNullOrEmpty(email))
+                return NotFound();
+
+            var result = await _authenticationService.ForgetPasswordAsync(email);
+
+            if (result.IsSuccess)
+                return Ok(result);
+
+            return BadRequest(result);
+        }
+
+        /// <summary>
+        /// ResetPassword
+        /// </summary>
+        /// <param name="model"></param>
+        /// <returns></returns>
+        [HttpPost("ResetPassword")]
+        public async Task<IActionResult> ResetPassword([FromForm] ResetPasswordViewModel model)
+        {
+            if (ModelState.IsValid)
+            {
+                var result = await _authenticationService.ResetPasswordAsync(model);
+
+                if (result.IsSuccess)
+                    return Ok(result);
+
+                return BadRequest(result);
+            }
+
+            return BadRequest("Some properties are not valid");
+        }
+
+        /// <summary>
+        /// Get user by username
+        /// </summary>
+        /// <param name="username"></param>
+        /// <returns></returns>
+        [Authorize(Roles = "Admin")]
+        [HttpGet]
+        public async Task<ActionResult<UserResponseModel>> GetUserByUserNameAsync(string username)
+        {
+            return Ok(await _authenticationService.GetUserByUserName(username));
+        }
+
+        /// <summary>
+        /// Get user by id
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        [Authorize(Roles = "Admin")]
+        [HttpGet]
+        public async Task<ActionResult<UserResponseModel>> GetUserByIdAsync(int id)
+        {
+            return Ok(await _authenticationService.GetUserById(id));
+        }
+
+        /// <summary>
+        /// Get all users
+        /// </summary>
+        /// <returns></returns>
+        [Authorize(Roles = "Admin")]
+        [HttpGet]
+        public async Task<ActionResult<IEnumerable<UserResponseModel>>> GetUsersAsync()
+        {
+            return Ok(await _authenticationService.GetUsers());
+        }
+
+        /// <summary>
+        /// Get agency by agency name
+        /// </summary>
+        /// <param name="agencyName"></param>
+        /// <returns></returns>
+        public async Task<ActionResult<AgencyResponseModel>> GetAgencyByNameAsync(string agencyName)
+        {
+            return Ok(await _authenticationService.GetAgencyByName(agencyName));
+        }
+
+        /// <summary>
+        /// Get agency by agency id
+        /// </summary>
+        /// <param name="id"></param>
+        /// <returns></returns>
+        public async Task<ActionResult<AgencyResponseModel>> GetAgencyByIdAsync(int id)
```

Comment:
Missing endpoints, that will allow admin manage employees - adding new employee to agency, remove/suspend existing employee, edit existing employee Data (Name, job-title etc)